### ISSUE TITLE
Error: connect ECONNREFUSED 127.0.0.1:80

### ISSUE THREAD

**Issue by thomas-gosnet** [spencermountain/wtf_wikipedia#50]
Hi, I tried to use wtf_wikipedia but me and another person have this issue

```
PS C:\Users\Jormungand\Workspace\unicorngame> wikipedia Toronto Blue Jays
{ Error: connect ECONNREFUSED 127.0.0.1:80
    at Object.exports._errnoException (util.js:1012:11)
    at exports._exceptionWithHostPort (util.js:1035:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1080:14)
  code: 'ECONNREFUSED',
  errno: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 80,
  response: undefined }
{
  "type": "page",
  "text": {},
  "categories": [],
  "images": [],
  "infobox": {},
  "infobox_template": "",
  "tables": [],
  "translations": {}
}
```

Also, I have
npm 4.0.1
node 6.3.1
Test with window an Mac OS
If anyone have a small idea, it would be of great help to me.

**Comment by thomas-gosnet** [comment]
Ok I found it's because you don't specify in fetch_text.js that the url begin by https

**Comment by spencermountain** [comment]
oh, whoa. really?
thanks thomas.
yeah, i should fix that, if it didn't work. I'm a bit confused - is it because you're using a non-enwikipedia config?

**Comment by spencermountain** [comment]
either way, both api calls should work, shouldn't they? Unless wikipedia changed some kind of setting?
i just checked the same command, and couldn't reproduce it.

**Comment by thomas-gosnet** [comment]
What's enwikipedia config, I don't know this one

**Comment by thomas-gosnet** [comment]
Oh a last thing, the API is case sensitive if you make a search with ```The legend of zelda``` it doesn't work, you must have ```The Legend of Zelda```

**Comment by spencermountain** [comment]
oooooh, i see what you mean!
yes - this is my bad. I will fix this asap.
yeah, my bad!

**Comment by spencermountain** [comment]
hey, think I fixed this - update to `v0.8.4` and try it again?

**Comment by thomas-gosnet** [comment]
I didn't have any time to work with this, but it seems working fine
